tests/src/generated/resources/data/neotests_reloadable_reg_data_maps/data_maps/loot_table/effect_grant.json

```diff
@@ -0,0 +1,13 @@
+{
+  "values": {
+    "minecraft:blocks/copper_block": {
+      "duration": 100,
+      "id": "minecraft:nausea",
+      "neoforge:cures": [
+        "protected_by_totem",
+        "milk"
+      ],
```

Comment:
This is different from the 1.21.3 version of this PR, where this section is absent there. Why?

Reply:
1.21.3 does not have the cure system